### PR TITLE
Snagging | SPACE / Retuneable Media Element

### PR DESCRIPTION
**Pull request by alexxpetrov**
Closes:
- https://github.com/sparkletown/internal-sparkle-issues/issues/1835

+ Fixed "custom embed url" Input value. We can now use it properly

image

### CODE REVIEW

File: src/components/attendee/RetunableMediaElement/components/Tuner.tsx [hunk 1]
```diff
@@ -55,6 +55,7 @@ export const Tuner: React.FC<TunerProps> = ({
       });
     } else if (selectedSource === RetunableMediaSource.embed) {
       const value = embedUrlInputEl?.current?.value;
+      console.log(value);
```
Comment (denisdimitrov) [inline src/components/attendee/RetunableMediaElement/components/Tuner.tsx]: .

File: src/components/attendee/VideoComms/VideoCommsParticipant.tsx [hunk 2]
```diff
@@ -105,20 +97,6 @@ export const VideoCommsParticipant: React.FC<VideoCommsParticipantProps> = ({
           )}
         </div>
       </div>
-      {screenshareTrack?.enabled && (
```
Comment (denisdimitrov) [inline src/components/attendee/VideoComms/VideoCommsParticipant.tsx:108]: Is this removed because of the Retunable component?

**Review by denisdimitrov — APPROVED** [review]
LGTM!

2 non blocking comments.

File: src/components/attendee/VideoComms/VideoCommsParticipant.tsx (hunk 2 above)
Comment (alexxpetrov) [reply src/components/attendee/VideoComms/VideoCommsParticipant.tsx:108]: Yep, I'm removing this, so the screen share is not displayed in the webcam grid. Instead it's going to be available as the "Media" element

File: src/components/attendee/RetunableMediaElement/components/Tuner.tsx (hunk 1 above)
Comment (alexxpetrov) [reply src/components/attendee/RetunableMediaElement/components/Tuner.tsx]: [769ce22](https://github.com/sparkletown/sparkle/pull/3035/commits/769ce226933fe1d0007860505b6fa93625d10f70)

**Pull request merged**